packages/types/src/metadata/PortableRegistry/PortableRegistry.ts

```diff
@@ -83,7 +83,10 @@ const PATHS_ALIAS = splitNamespace([
   'ink::env::types::*',
   'ink::primitives::types::*',
   'ink_env::types::*',
-  'ink_primitives::types::*'
+  'ink_primitives::types::*',
+  // noir
+  'np_runtime::accountname::AccountName',
+  'np_runtime::universaladdress::UniversalAddress'
```

Comment:
What is the background behind these 2 changes?

Reply:
I updated PR description. I thought that my use case had been explained by the connected issue. ;)

Reply:
Umpf, my bad. In this case I blame GH, since upon clicking it took me to the commits, not the description. Will read above :)